### DISCUSSION TITLE
emitWarning will trigger webpack cache warning

### DISCUSSION THREAD

**Discussion by PengBoUESTC** [webpack/webpack#16636] [Q&A]
# Bug report

**What is the current behavior?**
in `postcss-loader,`, it use emitWarning to report warning msg, but this lead to a `webpack cache` warning like this
`[webpack.cache.PackFileCacheStrategy] Skipped not serializable cache item`

is this a problem or not ? 

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Other relevant information:**
webpack version:  5.69.0
Node.js version: 
Operating System: 
Additional tools:

**Comment by alexander-akait** [comment]
Yes, any warnings and errors don't allow to cache module (by design), my recomendation - solve a warning
